test/e2e/tests/helper.py

```diff
@@ -136,4 +136,18 @@ def list_layer_versions(self, layer_name:str) -> list:
             return resp
         except Exception as e:
             logging.debug(e)
-            return None
\ No newline at end of file
+            return None
+    
+    def get_function_event_invoke_config(self, function_name:str) -> dict:
+        try:
+            resp = self.lambda_client.get_function_event_invoke_config(
+                FunctionName = function_name,
+                # Qualifier = qualifier
+            )
+            return resp
+        except Exception as e:
+            logging.debug(e)
+            return None
+    
+    def function_event_invoke_config_exists(self, function_name: str) -> bool:
+        return self.get_function_event_invoke_config(function_name) is not None
```

Comment:
@Vandita2020 I suggest to test `get_function_event_invoke_config` and see how it works - i don't see any specific errors returned by the controller.
Also another route, would be to test the controller locally and see what are the function `status.Conditions` when you try to set an `EventInvokeConfig`